### DISCUSSION TITLE
undefined

### DISCUSSION THREAD

**Discussion by giscus** [giscus/giscus#1445] [General]
# undefined

VAEs构建生成式时间序列建模技术是一个关键领域,其应用远不止于模拟数据。我们这个方法可以适用于数据插补、异常检测和预测等应用。 通过使用一维卷积层、策略性步幅、灵活的时间输入和季节性先验,可以构建一个能够复制时间序列中复杂模式的VAE。让我们合作完善时间序列建模的最佳实践。 生成式时间序列在异常检

https://walefall.github.io/2024/09/05/%E7%94%9F%E6%88%90%E5%BC%8F%E6%97%B6%E9%97%B4%E5%BA%8F%E5%88%97/

**Comment by walefall** [comment]
测试

**Comment by walefall** [comment]
是大
